src/settings/LocationLocations/LocationForm/ServicePointsFields.js

```diff
@@ -134,10 +141,10 @@ class ServicePointsFields extends React.Component {
       <>
         <FieldArray
           addLabel={
-            Object.keys(this.list).length > 1 ?
-              <Icon icon="plus-sign">Add service point</Icon> :
-              ''
-          }
+                canAddMoreServicePoints ?
+                  <Icon icon="plus-sign">Add service point</Icon> :
+                  ''
+              }
```

Comment:
Looks like this part of the code moved aside. Please fix this

Reply:
fixed